packages/graphql-server/tsconfig.json

```diff
@@ -1,19 +1,25 @@
 {
-  "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "baseUrl": "./",
-    "sourceRoot": "/",
-    "paths": {
-      "@/*": ["src/*"],
-      "@reapit/cognito-auth": ["../cognito-auth/src"],
-      "@reapit/elements": ["../elements/src"]
-    }
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "target": "es6",
+    "jsx": "react",
```

Comment:
This package is now pointing to the production version of @reapit/elements which doesn't contain JSX code. I think this option seems to be redundant